src/renderer/dx/DxRenderer.cpp

```diff
@@ -147,39 +149,38 @@ DxEngine::~DxEngine()
                               // D3D11_CREATE_DEVICE_DEBUG |
                               D3D11_CREATE_DEVICE_SINGLETHREADED;
 
-    D3D_FEATURE_LEVEL FeatureLevels[] = {
-        D3D_FEATURE_LEVEL_11_1,
-        D3D_FEATURE_LEVEL_11_0,
-        D3D_FEATURE_LEVEL_10_1,
-        D3D_FEATURE_LEVEL_10_0,
-        D3D_FEATURE_LEVEL_9_1,
-    };
+    std::array<D3D_FEATURE_LEVEL, 5> FeatureLevels;

```

Comment:
we should be able to initializer-list initialize this even as an std::array. I'm averse to using a hundred ats for this.

Reply:
Me too but I didn't think initializer would work. Trying it.